test/controllers/LineageExplorerControllerSpec.scala

```diff
@@ -91,13 +93,28 @@ class LineageExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
 
     }
   }
+
   "The lineage from context controller" should {
-    "return an empty list if the id of the node is not a deployernode" in {
-      val result = lineageController.lineageFromContext( 0L ).apply( fakerequest )
-      val content = contentAsJson( result ).as[List[JsObject]]
+    "return a 404 NotFound if the node does not exist" in {
+      val id = g.V().has( Constants.TypeKey, "resource:file" ).asScala.toList.head.id
```

Comment:
you swapped with the next one. Here the node exists

Reply:
Will fix, thanks